app/src/main/java/com/android_academy/covid_19/providers/InfectedLocationsWorker.kt

```diff
@@ -38,12 +38,15 @@ class InfectedLocationsWorker(
             locationRepo.getLocation()
 
             // get my cached locations for last 14 days
+            Timber.d("[InfectedLocationsWorker], doWork(): locationRepo.getUserLocationsAsync()")
             val myLocations: List<UserLocationModel> = locationRepo.getUserLocationsAsync()
 
             // get infected locations from server
+            Timber.d("[InfectedLocationsWorker], doWork(): infectionDataRepo.getInfectionLocationsAsync")
             val infectedLocations = infectionDataRepo.getInfectionLocationsAsync(0, 0)
 
             // run colliding algorithm
+            Timber.d("[InfectedLocationsWorker], doWork(): collisionMatcher.isColliding")
```

Comment:
Why do you need this "[InfectedLocationsWorker]", you have automatically InfectedLocationsWorker TAG in the log